bool arm
int8 algorithm
int8 trajectory_mode 
float64 x
float64 y
float64 z 
float64 yaw
